src/reducers/layout.js

```diff
@@ -38,25 +13,6 @@ const INITIAL_STATE = {
   showSideMenu: true,
   lab: {
```

Comment:
Is it necessary to have this structure lab.stageDimensions? Maybe we can get rid of `lab`.

Reply:
Good idea 👍